src/Lean/Elab/Command.lean

```diff
@@ -557,19 +557,21 @@ private def mkMetaContext : Meta.Context := {
 
 open Lean.Parser.Term in
 /-- Return identifier names in the given bracketed binder. -/
-def getBracketedBinderIds : Syntax → Array Name
-  | `(bracketedBinderF|($ids* $[: $ty?]? $(_annot?)?)) => ids.map Syntax.getId
-  | `(bracketedBinderF|{$ids* $[: $ty?]?})             => ids.map Syntax.getId
-  | `(bracketedBinderF|[$id : $_])                     => #[id.getId]
-  | `(bracketedBinderF|[$_])                           => #[Name.anonymous]
-  | _                                                 => #[]
-
-private def mkTermContext (ctx : Context) (s : State) : Term.Context := Id.run do
+def getBracketedBinderIds : Syntax → CommandElabM (Array Name)
```

Comment:
@semorrison I'm afraid this signature change will break exactly one line in Mathlib

Reply:
Fixed!